doc/builtin_entities.txt

```diff
@@ -0,0 +1,96 @@
+# Builtin Entities
+Minetest registers two entities by default: Falling nodes and dropped items.
+This document describes how they behave and what you can do with them.
+
+## Falling node (`__builtin:falling_node`)
+
+Is created when a node with the special group `falling_node=1` starts to fall.
+Falling nodes can also be created artificially with
+`minetest.spawn_falling_node`.
+
+Spawning a falling node with `/spawnentity` won't work. 
+
+Behaviour:
+
+* Falls down in a straight line (gravity = `movement_gravity` setting)
+* It collides with physical objects and `walkable` nodes
+* If the node group `float=1` is set, it also collides with liquid nodes
+* When it hits a solid (=`walkable`) node, it will try to place itself as a
+  node, replacing the node above
+    * If the destination node is not `buildable_to`, the *destination node*
+      drops as an item
+    * If the falling node fails to place itself, it drops as an item
+
+### Entity fields
+
+* `set_node(self, node, meta)`: Call this function to set the node type that
+  is falling. `node` and `meta` are explained below. The `meta` argument
+  is optional.
+* `node`: Node table of the node (`name`, `param1`, `param2`) that this
+  entity represents. Read-only.
+* `meta`: Node metadata of the falling node. Will be used when the falling
+  nodes tries to place itself as a node. Read-only.
+
+### Rendering / supported nodes
+
+Falling nodes try their best to look as close as possible to the original node.
```

Comment:
`try their best` sounds cute in some way. However, I'd expect different from a documentation file.

```suggestion
Falling nodes have visuals close as possible to the original node.
```